### ISSUE TITLE
Bug Personal portfolio story #7

### ISSUE THREAD

**Issue by tegardp** [freeCodeCamp/testable-projects-fcc#826]
#### Issue Description

I am failing test 7, The navbar should contain at least one link that I can click on to navigate to different sections of the page. I have three links and they are not working. I am new to coding and admit I could be wrong but I have been over the a tags in the navbar and the id names for each section..I think there is an issue with the tester?

#### Browser Information

* Browser Name, Version: Chrome
* Operating System: Mojave
* Mobile, Desktop, or Tablet: Desktop

#### Your Code / Link to Your Pen

```
 https://codepen.io/shinxin/pen/ywXZvm
 
 
```

**Comment by tbushman** [comment]
@tegardp Thank you. This seems like a bug. Note that you have a duplicate href in your navbar, `contact` , but that the test still fail after fixing that. Thanks.

**Comment by ValeraS** [comment]
@tegardp If set `height:100%` and `overflow-x:hidden` on html selector then window scroll functions and properties do not work. To pass the test do not set both these properties at the same time on html selector.

**Comment by zelr0x** [comment]
My resume project had a similar problem (same test case) but mine was caused by `fullpage.js`. I was able to pass the test by setting `scrollBar:  true` or `autoScrolling: false` (the last one kinda breaks the purpose of the lib) in fullpage constructor. 

I wanted to create a new bug report but then I found this issue is still open. I hope this helps somebody in some way. My pen is `https://codepen.io/zelr0x/pen/axVpbV`
